backend/gce/gce.go

```diff
@@ -56,6 +63,11 @@ func init() {
 	backend.Register("gce", New)
 }
 
+// EnvGCENetworkUseAliasIP is an environment variable that determines whether to use static routes or alias IPs
+// When "true", flannel adds alias IPs to instances
+// When any other value or unset, flannel inserts static routes
```

Comment:
nitpick: ParseBool accepts more than just "true": https://golang.org/pkg/strconv/#ParseBool

nitpick: description doesn't match implementation, due to returning an error on ParseBool other values could cause errors instead of going into the static routes code.

Reply:
You're right, this needs fixed because the code will throw an error if the environment variable isn't provided. I originally was just checking `"true"` but changed it during internal review. Let me fix it and send this through a full testing cycle just in case.

Reply:
I committed some changes and tested with (1) `FLANNEL_GCE_NETWORK_USE_ALIAS_IP=true` and (2) no environment variable and it worked as expected.